pfcpiface/parse-far.go

```diff
@@ -138,6 +140,8 @@ func (f *far) parseFAR(farIE *ie.IE, fseid uint64, upf *upf, op operation) error
 				f.tunnelIP4Src = ip2int(upf.accessIP)
 			} else if f.dstIntf == ie.DstInterfaceCore {
 				f.tunnelIP4Src = ip2int(upf.coreIP)
+			} else {
```

Comment:
What case is this?

Reply:
this is in case of SGI/LAN coming as destination interface.

Reply:
Why does SGILAN need tunneling?